dojo-starter/src/systems/actions.cairo

```diff
@@ -3,7 +3,8 @@ use dojo_starter::models::game::Game;
 // define the interface
 #[dojo::interface]
 trait IActions {
-    fn start_game() -> Game;
+    fn start_game(world:IWorldDispatcher) -> Game;
+    fn end_game(game:Game) -> None;
```

Comment:
you don't have to return anything in the case when we don't have anything

Reply:
okay